webpack/simple_named_modules.js

```diff
@@ -0,0 +1,33 @@
+/*
+  Simple Named Modules Plugin
+  Strips relative path up to and including node_modules/ from the module ID.
+  This allows for consistent module IDs when building webpack bundles from
+  differing base paths relative to the node_modules directory.
+
+  Based on NamedModulesPlugin by Tobias Koppers @sokra, originally licensed under
+  MIT License: http://www.opensource.org/licenses/mit-license.php
+*/
+"use strict";
+
+class SimpleNamedModulesPlugin {
+  constructor(options) {
+    this.options = options || {};
+  }
+
+  apply(compiler) {
+    compiler.plugin("compilation", (compilation) => {
+      compilation.plugin("before-module-ids", (modules) => {
+        modules.forEach((module) => {
+          if(module.id === null && module.libIdent) {
+            module.id = module.libIdent({
+              context: this.options.context || compiler.options.context
+            });
+            module.id = module.id.slice(module.id.indexOf('node_modules') + 13)
```

Comment:
what does 13 mean?

Reply:
length('node_modules/')

Reply:
I'm not sure if module IDs can also be our `webpack` directory.

Reply:
I don't think they should, since we don't export anything from bundle.js. 
we could use lastIndexOf('../') instead and leave the node_modules path if we want to be certain.